### DISCUSSION TITLE
Can µManager be used for Live Video, Manual counts and measures

### DISCUSSION THREAD

**Discussion by telpochyaotl** [micro-manager/mmCoreAndDevices#424] [General]
I hope it's ok to ask this here. Can µManager be used to perform live nematode scans, protozoa counts, bacterial counts and filaments measurements? My microscope came with ToupView and only ToupLite for the Mac/Linux, It's good but could be better, specially if I could customize µManager to do length measurements in FOV units...  So while I look at how to install it and test it.... I have not seen YouTube videos for example, that show µManager with a live, unlimited view of the Microscope Camera... Will it be suitable for this use case? Thank you so much!

**Comment by marktsuchida** [comment]
Hi @telpochyaotl, the best place for these kinds of questions is the [Scientific Community Image Forum](https://forum.image.sc/), with tag `micro-manager`.

I think some models of ToupTek cameras might(?) work in Micro-Manager via the AmScope device adapter, but only on Windows.
